### PR TITLE
IGNITE-14741

### PR DESCRIPTION
**Pull request by vldpyatkov**

### CODE REVIEW

File: modules/affinity/src/main/java/org/apache/ignite/internal/affinity/AffinityManager.java [hunk 1]
```diff
@@ -88,7 +88,9 @@ public AffinityManager(
         metaStorageMgr.registerWatchByPrefix(new ByteArray(INTERNAL_PREFIX), new WatchListener() {
             @Override public boolean onUpdate(@NotNull WatchEvent watchEvt) {
                 for (EntryEvent evt : watchEvt.entryEvents()) {
-                    String tabIdVal = evt.newEntry().key().toString().substring(INTERNAL_PREFIX.length());
+                    String keyAsString = new String(evt.newEntry().key().bytes(), StandardCharsets.UTF_8);
```
Comment (sanpwc) [inline modules/affinity/src/main/java/org/apache/ignite/internal/affinity/AffinityManager.java:91]: I'd rather use 
```
new ByteArray(evt.newEntry().key().bytes()).toString()
```
in order to hide StandardCharsets.UTF_8

File: modules/affinity/src/test/java/org/apache/ignite/internal/affinity/AffinityManagerTest.java [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+     * @return List of root configuration keys.
+     */
+    private static List<RootKey<?, ?>> rootConfigurationKeys() {
+        return Arrays.asList(
+            TablesConfiguration.KEY
+        );
+    }
+
+    @Test
+    public void testCalculatedAssignment() {
```
Comment (sanpwc) [inline modules/affinity/src/test/java/org/apache/ignite/internal/affinity/AffinityManagerTest.java:133]: javadoc

File: modules/affinity/src/main/java/org/apache/ignite/internal/affinity/AffinityManager.java (hunk 1 above)
Comment (vldpyatkov) [reply modules/affinity/src/main/java/org/apache/ignite/internal/affinity/AffinityManager.java]: Done.

File: modules/affinity/src/test/java/org/apache/ignite/internal/affinity/AffinityManagerTest.java (hunk 2 above)
Comment (vldpyatkov) [reply modules/affinity/src/test/java/org/apache/ignite/internal/affinity/AffinityManagerTest.java:136]: Wrote a description.

File: modules/table/src/test/java/org/apache/ignite/internal/table/TableManagerTest.java [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+        catch (Exception e) {
+            LOG.error("Failed to bootstrap the test configuration manager.", e);
+
+            fail("Failed to configure manager [err=" + e.getMessage() + ']');
+        }
+    }
+
+    @Disabled("Static configuration issue.")
+    @Test
+    public void testStaticTableConfigured() {
```
[the remark this answers is not in the document]
Comment (vldpyatkov) [reply modules/table/src/test/java/org/apache/ignite/internal/table/TableManagerTest.java:167]: Done.

**Pull request closed without merging**